qemu/tests/cfg/format_disk.cfg

```diff
@@ -13,7 +13,9 @@
     format_cmd = "cd /dev && for i in `ls | egrep [shv]db`;do yes |mkfs.ext3 $i; done"
     list_disk_cmd = ""
     set_online_cmd = ""
+    show_dev_cmd = "cd /dev && ls | egrep [shv]db"
     mount_cmd =  "mkdir -p /media && cd /dev && ls | egrep [shv]db | xargs -I dev mount -t ext3 dev /media"
+    show_mount_cmd = "mount | grep -Eo '/dev/[shv]db'"
```

Comment:
As this case should support Windows guest, please provide related command for windows guest too.